examples/create-react-app/config-overrides.js

```diff
@@ -1,7 +1,10 @@
 const ReactRefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
 const { addBabelPlugin, addWebpackPlugin, override } = require('customize-cra');
 
+const webpackEnv = process.env.NODE_ENV;
```

Comment:
Just a single variable would be fine?

```suggestion
const isDevelopment = process.env.NODE_ENV === 'development';
```